src/installer/tests/TestUtils/RuntimeConfig.cs

```diff
@@ -83,9 +83,9 @@ internal JObject ToJson()
                 return frameworkReference;
             }
 
-            internal static Framework FromJson(JObject jobject)
+            internal static Framework FromJson(JsonObject jobject)
             {
-                return new Framework((string)jobject["name"], (string)jobject["version"])
+                return new Framework(jobject["name"].ToString(), jobject["version"].ToString())
```

Comment:
@eiriktsarpalis, the reason for changing this line was given these two files:

```json
{  "name": "foo" }
```
```json
{  "name": false }
```
first one successfully casts to string but second one throws an exception about `'False' type` conversion to `System.String`.

If this is not an intentional decision then IMO, we should allow explicit cast to string on `JsonNode` for all JSON types.